policyengine_canada/parameters/gov/provinces/nt/tax/income/benefits/child_benefit/age_eligibility.yaml

```diff
@@ -0,0 +1,11 @@
+description: Northwest Territoriesa provides a benefit amount for children below this age under the child benefit.
```

Comment:
description: Northwest Territories provides a child benefit to filers below this age threshold.